stack/docker-compose.yml

```diff
@@ -0,0 +1,105 @@
+name: refarch-stack
+
+services:
+  # s3
+  minio:
+    image: quay.io/minio/minio:latest
+    command: server /data --console-address ":9001"
+    environment:
+      MINIO_ROOT_USER: minio
+      MINIO_ROOT_PASSWORD: Test1234
+    ports:
+      - '9000:9000'
+      - '9001:9001'
+    volumes:
+      - ./minio:/data
+    networks:
+      - internal
```

Comment:
Could you please add restart policy, security opts and additional health check (if not provided by the image itself) to the additional containers you provided in comparison to the default stack in https://github.com/it-at-m/refarch-templates/blob/main/stack/docker-compose.yml

Reply:
Done